io.gitlab.celleron56.libretile.yaml

```diff
@@ -0,0 +1,93 @@
+app-id: io.gitlab.celleron56.libretile
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: libretile.sh
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+
+
+modules:
+  - name: libretile
+    buildsystem: simple
+    
+    build-commands:
+       - mkdir /app/files
+       - mv * /app/files
+       - echo || ls
+       - echo starting build 
+       - echo "python3  /app/files/main.py" >> libretile.sh
+       - ln -s libretile.sh libretilelink.sh
+       - install -D libretilelink.sh /app/bin/libretile.sh
+    sources:
+      - type: git
+        url: https://gitlab.com/celleron56/libretile.git
+        branch: stable
```

Comment:
never a branch. you need a specific commit and/or tag.

Reply:
ok     the branch is stable though :P  
(i organize my repos in a way that each version is a branch )

i'll see about adding tags